fastai/callbacks/lr_finder.py

```diff
@@ -24,11 +25,16 @@ def on_train_begin(self, pbar, **kwargs:Any)->None:
 
     def on_batch_end(self, iteration:int, smooth_loss:TensorOrNumber, **kwargs:Any)->None:
         "Determine if loss has runaway and we should stop."
+        self.batch_num += 1
```

Comment:
Why are you not using `iteration`? It is the same thing (starts at 0 at the beginning of training and is incremented at each batch).

Reply:
Hello Sylvain, ha, glad to learn something new --- I wasn't aware what  `iteration` does.  Sure enough that's the piece to use.  Thank you!!  Upon digging I also learned about `epoch` and `num_batch` in the implicit parameters passed from `state_dict`.

I have updated the PR to use these three existing entries to do what I intended.